File: golang-weather/entities/weather_forecast.go
package entities

type WeatherForecast struct {
  Date string `json:"date" gorm:"column:date"`
  TemperatureC int `json:"temperaturec" gorm:"column:temperaturec"`
  TemperatureF int `json:"temperaturef"`
  City string `json:"city" gorm:"column:city"`
  Summary string `json:"summary" gorm:"column:summary"`
}

File: bun-weather/src/entities/weather-forecast.ts
import { Column, Entity, PrimaryColumn } from "typeorm"

@Entity("WeatherForecast")
export class WeatherForecast {
  @PrimaryColumn({ name: "id", type: "int" })
  id!: number
  @Column({name: "date", type: "date"})
  date!: Date
  @Column({name: "temperaturec", type: "integer"})
  temperatureC!: number
  temperatureF!: number
  @Column({name:"summary", type: "varchar"}) summary?: string
  @Column({name: "city", type: "varchar"}) city?: string
}
